src/modules/esl-forms/esl-select/core/esl-select-dropdown.ts

```diff
@@ -49,6 +50,11 @@ export class ESLSelectDropdown extends ESLToggleable {
   }
 
   protected onShow(params: ESLToggleableActionParams): void {
+    if (this.activator) {
+      const renderer = ESLTraversingQuery.first('esl-select-renderer', null, this.activator) as ESLSelectRenderer | null;
+      if (renderer) this.activator = renderer;
+    }
```

Comment:
I don't think it's a good thing to have a coupe source to feel this property...
Propose to find the reason of losing activator property in the original mechanism.

Reply:
Activator lose should be fixed in #1448